src/main/java/com/example/macro/user/service/UserService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.macro.user.service;
+
+import com.example.macro.user.document.User;
+
+public interface UserService {
```

Comment:
Do we really need an interface if there's only going to be a single implementation defined?